spec/classes/pulp_crane_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+
+describe 'pulp::crane' do
+
+ context 'on redhat' do
+    context 'with parameters' do
+      let :pre_condition do
```

Comment:
Since you're testing this class I believe you can use ```let :params``` instead of a pre condition.